app/helpers/application_helper.rb

```diff
@@ -1,9 +1,70 @@
 module ApplicationHelper
+  ALERT_KIND_MAPPING = {
+    notice: "alert-info",
+    success: "alert-success",
+    error: "alert-error"
+  }
+
+  ALERT_ICON_MAPPING = {
+    notice: "information-circle",
+    success: "check-circle",
+    error: "x-circle"
+  }
+
   def page_rel(page)
     if page.prev?
       "next"
     elsif page.prev?
       "prev"
     end
   end
+
+  def back_to(text = t("back"), path, **attributes)
+    classes = [ class_names("flex items-center gap-2"), attributes.delete(:class) ].compact.uniq.join(" ")
+
+    link_to path, class: classes, **attributes do
+      lucide_icon("move-left").concat(content_tag(:span, text, class: "link-text"))
+    end
+  end
+
+  def external_link_to(text, url = nil, **attributes, &block)
+    if block_given?
+      text = capture(&block)
+      url = text
+    end
+
+    classes = [ class_names("inline-flex items-center gap-2 flex-nowrap"), attributes.delete(:class) ].compact.uniq.join(" ")
```

Comment:
```suggestion
    classes = class_names("inline-flex items-center gap-2 flex-nowrap"), attributes.delete(:class)
```

De mémoire, le helper class_names fait le ménage pour toi et retourne une chaîne de caractères, tu n'as donc pas besoin de le faire manuellement

Reply:
oui en effet j'étais fatigué la lol